source/server/spongineer/commands.rst

```diff
@@ -60,6 +60,19 @@ Command                 Description                               Permission
 * /sponge save [-g] [-d dim|*] [-w world|*]
 * /sponge reload [-g] [-d dim|*] [-w world|*]
 
+**Command Conflicts**
+
+In cases of command conflict, Sponge provides a primary alias mechanism to specify which command is to be used.  For 
+example, Minecraft provides the `reload <https://minecraft.gamepedia.com/Commands#Summary_of_commands>`__ command and 
+Sponge provides the `reload <https://docs.spongepowered.org/stable/en/server/spongineer/commands.html>`__ command. To 
+specify which command to use, prefix it with minecraft or sponge and a ``:``. So, to use the Sponge reload command 
```

Comment:
Oh there is actually another one here, isn't it?

>  the Sponge reload command 

-> 

>  Sponge's reload command

Reply:
I like your version better.  I'll get an update up real quick.